### DISCUSSION TITLE
Issue with webhooks

### DISCUSSION THREAD

**Discussion by freyntjens** [Shuffle/Shuffle#1010] [Q&A] [answered]
Hi guys,

Currently having an issue with webhooks triggers: seams quite simple but I must be doing something wrong.

Have a very simple workflow as per bellow

Webhook trigger -> Shuffle tools (repeat back to me: "hello world")

Performing a "curl" in order to test the trigger

curl  -XPOST -k [https://<Shuffle>/api/v1/hooks/webhook_xxx-xxx-xxx](https://%3cShuffle%3e/api/v1/hooks/webhook_xxx-xxx-xxx) --data '{"test": "testing"}'
{"success": true, "execution_id": "xxx-xxx-xxx"}

As per Shuffle logs workflow should be executed:

2023/01/04 15:56:35 [INFO] Found ID xxxc-xxx-xxx for hook
2023/01/04 15:56:35 [INFO] Running POST execution with body of length 124 for workflow xxxc-xxx-xxx
2023/01/04 15:56:35 [INFO] Should start execution on node xxxc-xxx-xxx
2023/01/04 15:56:35 [INFO] Execution source is webhook for execution ID xxxc-xxx-xxx in workflow xxxc-xxx-xxx
2023/01/04 15:56:35 [INFO] Execution: xxxc-xxx-xxx should execute onprem with execution environment "Shuffle". Workflow: xxxc-xxx-xxx
2023/01/04 15:56:39 [DEBUG] Got workflow result from 172.18.0.4:53126 of length 42711.
2023/01/04 15:56:39 [INFO] Set workflowexecution based on new worker (>0.8.53) for execution xxxc-xxx-xxx. Actions: 1, Triggers: 1, Results: 1, Status: FINISHED
2023/01/04 15:56:39 [DEBUG] Got workflow result from 172.18.0.4:53126 of length 42711.
2023/01/04 15:56:39 [INFO] Set workflowexecution based on new worker (>0.8.53) for execution xxxc-xxx-xxx. Actions: 1, Triggers: 1, Results: 1, Status: FINISHED

While in shuffle GUI I don't have the result expected i.e. workflow running with "hello word" as output… simply nothing happens…

Am I doing something wrong? Sorry to bother you guys for an apparently simple question but, a ny help would be highly appreciated.

Thanks,

Fred

**Comment by frikky** [comment, marked as answer]
Heyo!

It does indeed look like the webhook itself is working. Do you not see the executions in the workflow itself when looking at workflow executions (click the running person button on the bottom)?
All this kind of stuff runs in the background on purpose, but we are working on helping you understand better whether any executions have happened from webhooks.

**Reply by freyntjens** [reply]
Hi frikky,

Thanks for the swift reply!!!
It does work... I knew I was being dumb...
it runs in background and triggers correctly the shuffle tools app.
Thanks a lot mate!!!

Regards,

Fred
